pkgs/tools/networking/imapsync/default.nix

```
@@ -1,20 +1,19 @@
-{ lib
-, fetchFromGitHub
-, makeWrapper
-, perl
-, perlPackages
-, stdenv
+{
+  lib,
+  fetchurl,
+  makeWrapper,
+  perl,
+  perlPackages,
+  stdenv,
 }:
 
 stdenv.mkDerivation rec {
   pname = "imapsync";
-  version = "2.229";
+  version = "2.290";
 
-  src = fetchFromGitHub {
-    owner = "imapsync";
-    repo = "imapsync";
-    rev = "imapsync-${version}";
-    sha256 = "sha256-nlNePOV3Y0atEPSRByRo3dHj/WjIaefEDeWdMKTo4gc=";
+  src = fetchurl {
+    url = "https://imapsync.lamiral.info/dist/old_releases/2.290/imapsync-${version}.tgz";
```

```suggestion
    url = "https://imapsync.lamiral.info/dist/old_releases/${version}/imapsync-${version}.tgz";
```
Also would benefit from using `finalAttrs` so people can write `src.overrideAttrs (prev: { hash = "<new hash>" })` without specifying url again